tools/report-converter/README.md

```diff
@@ -21,6 +21,7 @@ a CodeChecker server.
 * [Pyflakes](#pyflakes)
 * [Markdownlint](#markdownlint)
 * [Coccinelle](#coccinelle)
+* [Smatch](#smatch)
```

Comment:
Please also update the usage section: https://github.com/Ericsson/codechecker/blob/19420f0495fac3878bffaf8488f21614cad1b841/tools/report-converter/README.md#usage

Reply:
I see that even coccinelle isnt included that well here as well along with smatch?

Reply:
You can update the doc with the coccinelle support in this patch too.